script.js

```diff
@@ -14,3 +205,67 @@ backMenu.forEach((element) => element.addEventListener('click', () => {
 }));
 
 /* ---------------------hamburger code above----------------------------- */
+
+/* ---------------------Windows popup code below----------------------------- */
+const body = document.querySelector('body');
+const view = document.querySelectorAll('.view-project');
+view.forEach((element) => element.addEventListener('click', () => {
+  const show = document.createElement('div');
+  const detailPop = document.createElement('div');
+  show.className = 'show';
+  detailPop.className = 'pop';
+  detailPop.innerHTML = `
+                      <div class="popup-header">
+                        <h2 class="desktop" id="desktop">Keeping track of hundreds
+                         of components website</h2>
+                         <h2 class="mob-title">Multi Post Stories</h2>
+                        <span id="close">&times;</span>
+                      </div>
+                        <div class="popup-button">
+                          <button id="lists">HTML</button>
+                          <button id="lists">Boostrap</button>
+                          <button id="lists">Ruby on rails</button>
+                        </div>
+                      <div class="popup-body">
+                        <img class="desktop" src="images/aya.png" alt="">
+                        <img class="mob" src="images/snapsmall.png" alt="">
+                        <div class="pop-details">
+                          <p>
+                          Lorem Ipsum is simply dummy text of the
+                          printing and typesetting industry.
+                          Lorem Ipsum has been the industry's
+                          standard dummy text ever since the 1500s,
+                          when an unknown printer took a
+                           galley of type and scrambled it 1960s.
+                          </p>
+
+                          <p class="desktop">
+                          Lorem Ipsum is simply dummy text of the
+                          printing and typesetting industry.
+                          Lorem Ipshum has been the industry's
+                          standard dummy text ever since the 1500s,
+                          when an unknown printer took a
+                           galley of type and scrambled it 1960s.
+                          </p>
+                         
+                          <div class="popup-button">
+                            <button class="popup-btn"><a href="">See live  <img class="link-img" src="images/link.png" />
+                            </a></button>
+                            <button class="popup-btn git"><a href="">See Source <img class="link-img" src="images/vector.png" />
```

Comment:
- 🙏Please fix this issue - the icon is not showing🙄. It seems that your image👩‍🦳 file is named 'Vector.png', but you are trying to reference it as 'vector.png' with a lowercase 'v'. Updating the reference to match the actual file📜 name. This will enable your icon to be shown in your portfolio.